exercises/armstrong-numbers/armstrong_numbers_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'minitest/autorun'
+require_relative 'armstrong_numbers'
+
+class ArmstrongNumbersTest < Minitest::Test
+  def test_one_digit_numbers_are_armstrong_numbers
+    assert ArmstrongNumbers.is_valid?(1)
+    assert ArmstrongNumbers.is_valid?(6)
+    assert ArmstrongNumbers.is_valid?(9)
+  end
+
+  def test_two_digit_numbers_aren_t_armstrong_numbers
+    refute ArmstrongNumbers.is_valid?(11)
+    refute ArmstrongNumbers.is_valid?(89)
+    refute ArmstrongNumbers.is_valid?(44)
+  end
+
+  def test_three_digit_numbers_that_are_the_armstrong_numbers
```

Comment:
`test_three_digit_numbers_that_are_the_armstrong_numbers` Should this be `test_three_digit_numbers_that_are_armstrong_numbers` ?

Same below.